Content.Server/GameObjects/EntitySystems/RoundEndSystem.cs

```diff
@@ -17,7 +17,7 @@ public class RoundEndSystem : EntitySystem
 
         private CancellationTokenSource _roundEndCancellationTokenSource = new CancellationTokenSource();
         public bool IsRoundEndCountdownStarted { get; private set; }
-        public int RoundEndCountdownTime { get; set; } = 5000;
+        public int RoundEndCountdownTime { get; set; } = 240000;
```

Comment:
See this is why I'd prefer this be stored as a `TimeSpan`, then you can just do
 ```cs
public TimeSpan RoundEndCountdownTime { get; set; } = TimeSpan.FromMinutes(4);
```

Reply:
Oh, that's much more convenient, alright! I changed it now.